src/ibl_to_nwb/ecephys/requirements.txt

```diff
@@ -0,0 +1,3 @@
+neuroconv[ecephys]==0.2.3
+ONE-api>=1.16.3
+ibllib>=2.19.0
```

Comment:
Can you remind me what this dependency was for? I think I also recall having to install it for something, but the docs seem to rely on just ONE for data transfers

Reply:
This is true when downloading, and for streaming they have a separate tool:
`from brainbox.io.spikeglx import Streamer`
This is packaged with `ibllib`.